src/Fixer/Casing/LowercaseConstantsFixer.php

```diff
@@ -24,8 +27,20 @@
  *
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  */
-final class LowercaseConstantsFixer extends AbstractFixer
+final class LowercaseConstantsFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
+    private $fixCallback;
```

Comment:
This misses (a doc block with) a typehint.

Reply:
I will fix today after work.

Reply:
Fixed.